src/components/network-chart/NetworkChartRow.js

```diff
@@ -268,15 +268,17 @@ class NetworkChartRow extends React.PureComponent<NetworkChartRowProps, State> {
     );
 
     return (
-      <section className={itemClassName}>
+      <section
```

Comment:
I'm not sure this should be a section, what do you think ?

Reply:
Probably not, it's an item in the list, not a section on its own.

Reply:
ok, I'll try to change it and if it doesn't break anything I'll keep it.